helpers/authentication.go

```diff
@@ -0,0 +1,19 @@
+package helpers
+
+import (
+	"log"
+
+	"golang.org/x/crypto/bcrypt"
+)
+
+// EncryptPassword encrypt given password, returns encrypted password and error
+func EncryptPassword(password string) (string, error) {
+	hash, err := bcrypt.GenerateFromPassword([]byte(password), bcrypt.MinCost)
+	if err != nil {
+		log.Println(err)
+
+		return "", err
+	}
+
+	return string(hash), err
```

Comment:
```suggestion
	return string(hash), nil
```
As we already check `if err != nil` above so now we can return nil here. It is more straightforward and easier to understand. What do you think?

Reply:
✅  Fixed in [f369cc8](https://github.com/carryall/go-google-scraper-challenge/pull/21/commits/f369cc8286f09fba63038ac5c648d96ce7739baf)